dashboard/test/factories/factories.rb

```diff
@@ -807,6 +807,13 @@
     description 'fake description'
   end
 
+  factory :vocabulary do
+    association :course_version
+    key 'word'
```

Comment:
in order to create more than one of these, I think we will want this to be a sequence, e.g. https://github.com/code-dot-org/code-dot-org/blob/4650698263c3086cbc3a1715881a6dd150f8dd7e/dashboard/test/factories/factories.rb#L806

also, just confirming my understanding that this will be an opaque id generated with something like `SecureRandom.uuid`, is that correct?

Reply:
I was originally thinking that `key` would be the `word` to start with but I think what you said makes more sense. We'll still need to prevent curriculum writers adding multiple definitions for the same word for now but I think that can be done in other ways. I'll add this sequence to the factory.

Reply:
should we consider just getting rid of the `key` column and having `word` be the key? sorry to add more options here.

Reply:
Don't be sorry -- I've been going back and forth between a bunch of these options! I thought about that but it's likely we will want to have multiple definitions for a handful of words. I thought about adding the key then but then we'd have to backfill everything already in the database.

Reply:
Just to summarize what we talked about offline -- we know we will need to have multiple definitions per word at some point, though not in the near term. We also need a unique identifier that is human-readable-ish in order to embed them in markdown. So, we will set the key to be the word in the near term and, in the future, generate human-readable alternate keys for duplicate definitions, like we do for resources